paasta_tools/list_marathon_service_instances.py

```diff
@@ -97,7 +97,7 @@ def get_desired_marathon_configs(soa_dir):
                 service=service,
                 line=str(errormsg),
                 component='deploy',
-                level='event',
+                level='debug',
```

Comment:
Looks like you based off your other branch instead of master - this is probably fine since we'll merge that one once you get on the GH org, but in the future you'll want to make sure your branches are based on master instead of other in-flight branches.

Reply:
Whoops my bad. Will do in the future.